index.html

```diff
@@ -1,62 +1,101 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title>Oksana Feterovskaya CTD lesson 4-2</title>
+        <title>Oksana Feterovskaya CTD lesson 4-5</title>
         <meta name="Oksana Feterovskaya" content="Oksana Fetter">  
         <meta charset="UTF-8">
-        <meta name="lesson 4-2" content="Code The Dream">
-        <meta name="design, website, html,DOM, lesson 4-2" content="HTML, CSS, JavaScript, DOM">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Neuton:wght@700&family=Noto+Serif:ital,wght@1,700&family=Roboto+Slab&display=swap" rel="stylesheet">
+        <meta name="lesson 4-5" content="Code The Dream">
+        <meta name="design, website, html,css layouts, lesson 4-5" content="HTML, CSS, JavaScript, DOM">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link rel = "stylesheet" href = "css/index.css">
+        <link rel = "stylesheet" href = "css/media.css">
     </head>
     <body>
-<nav>   
+<nav id="header">  
     <h1>Oksana Feterovskaya</h1>
     <h4>Web developer| Web designer| Web Technologies</h4>
+
+    <ul id="navbar">
+        <li><a href="#">About</a></li>
+        <li><a href="#experience">Experience</a></li>
+        <li><a href="#skills">Skills</a></li>
+        <li><a href="#connect">Connect</a></li>
+        <li><a href="#messagesForm">Leave a Message</a></li>
+    </ul>
 </nav>
-<section>
+
+<section class="section1">
     <h2 id = "text">About:</h2>
```

Comment:
Nitpick: Be careful about reusing ids for elements. Typically, when an id is assigned to an element it's unique from the rest of the elements on the page.

[Reference](https://www.w3schools.com/htmL/html_id.asp)
> The id attribute specifies a unique id for an HTML element. The value of the id attribute must be unique within the HTML document.